modules/EmergencyMedicine/acl/acl_upgrade.php

```diff
@@ -13,20 +13,165 @@
 
     '0.1.0' => function () {
```

Comment:
change to 0.2.0

In the version file we already in version 0.1.0, and meaning of the version in this file is to which version we need those new ACL , so with version 0.1.0 it will not run.

Reply:
ok